.eslintrc.json

```diff
@@ -5,6 +5,7 @@
   "overrides": [
     {
       "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "excludedFiles": "*.fixture.js",
```

Comment:
Fixtures should not be checked with `@nrwl/nx/enforce-module-boundaries` as it forces to use relative imports.